.github/workflows/debian_10_arm.yml

```diff
@@ -49,7 +49,7 @@ jobs:
   Docker:
     needs: Smart_CI
     if: "!needs.smart_ci.outputs.skip_workflow"
-    runs-on: aks-linux-16-cores-arm-docker-build
+    runs-on: aks-linux-4-cores-8gb-arm-docker-build
```

Comment:
for Smart CI it's not required to use ARM machine at all

Reply:
BTW, if ARM machines are more expensive than x86, we can perform cross-compilation on x86 for arm64 to reduce costs.

Reply:
> for Smart CI it's not required to use ARM machine at all

It's Docker job, not Smart CI, so that's expected

Reply:
> BTW, if ARM machines are more expensive than x86, we can perform cross-compilation on x86 for arm64 to reduce costs.

ARM machines are less expensive than x86 ones